packages/office-addin-manifest/src/commands.ts

```diff
@@ -129,28 +129,31 @@ export async function modify(manifestPath: string, command: commander.Command) {
 
 export async function validate(manifestPath: string, command: commander.Command) {
   try {
-    const validation: ManifestValidation = await validateManifest(manifestPath);
-
-    if (validation.isValid) {
-      console.log("The manifest is valid.");
-    } else {
-      console.log("The manifest is not valid.");
-    }
-    console.log();
-
-    if (validation.report) {
-      logManifestValidationErrors(validation.report.errors);
-      logManifestValidationWarnings(validation.report.warnings);
-      logManifestValidationInfos(validation.report.notes);
+    const manifest: manifestInfo.ManifestInfo = await manifestInfo.readManifestFile(manifestPath);
```

Comment:
In validate.ts, there is a readManifestFile() which is commented out. Seems better to uncomment it there.